dockerfiles/cli/scripts/cli.sh

```diff
@@ -174,6 +174,26 @@ cmd_config_post_action() {
     log "docker volume create --name=codenvy-postgresql-volume >> \"${LOGS}\""
     docker volume create --name=codenvy-postgresql-volume >> "${LOGS}"
   fi
+
+  if [ "${CHE_DEVELOPMENT_MODE}" = "on" ]; then
```

Comment:
Maybe we should have single folder with agents instead of adding small peace of code in this script on addition of each agent?